utils/index.js

```diff
@@ -1,6 +1,12 @@
-const { injectTemplates, shouldInjectTemplates } = require('./templates');
+const { getOneOther } = require('./get-one-other');
+const { jsonclone } = require('./jsonclone');
+const { glob } = require('./glob');
+const { shouldInjectTemplates, injectTemplates } = require('./templates');
 
 module.exports = {
-    injectTemplates,
-    shouldInjectTemplates
+    getOneOther,
+    jsonclone,
+    glob,
+    shouldInjectTemplates,
+    injectTemplates
 };
```

Comment:
Unlike ES modules, in commonJS exporting the utils like this means you will include the whole object if you require it through "utils" module. This means unused functions can not be tree-shaken.
I think it should be considered a bad practice in commonjs projects.